### PR TITLE
Implement cbuffer validate

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

**Comment by esc** [comment]
@FrancescAlted can I get a review on this?

File: blosc/blosc_extension.c [hunk 1]
```diff
@@ -539,6 +539,42 @@ PyBlosc_decompress(PyObject *self, PyObject *args)
 }
 
 
+PyDoc_STRVAR(cbuffer_validate__doc__,
+"cbuffer_validate(bytesobj) -- Check if compressed data is safe.\n\n"
+"Checks that it is safe to attempt decompression of compressed data.\n"
+"This does not guarantee that decompression will be successfull,\n"
```
Comment (FrancescAlted) [inline blosc/blosc_extension.c]: successfull -> successful

File: blosc/toplevel.py [hunk 2]
```diff
@@ -292,6 +292,36 @@ def get_cbuffer_sizes(bytesobj):
     return _ext.get_cbuffer_sizes(bytesobj)
 
 
+def cbuffer_validate(bytesobj):
+    """
+    validate_cbuffer(bytesobj)
+
+    Validate the cbuffer. Check that the cbuffer is safe to compress. Note
+    that, this does not guarantee that the blosc will be able to decompress the
```
Comment (FrancescAlted) [inline blosc/toplevel.py]: I'd remove the comma after the `that`, but YMMV ;)

File: blosc/blosc_extension.c (hunk 1 above)
Comment (esc) [reply blosc/blosc_extension.c]: fixed

File: blosc/toplevel.py (hunk 2 above)
Comment (esc) [reply blosc/toplevel.py]: fixed

**Comment by esc** [comment]
Review comments addressed, will merge once CI goes green.

**Pull request merged**